workos/event_objects/directory_user.py

```diff
@@ -0,0 +1,77 @@
+from typing import Optional, List
+from enum import Enum
+from workos.utils.types import JsonDict
+
+
+class DirectoryUserRole:
+    def __init__(self, attributes: JsonDict) -> None:
+        self.slug: str = attributes["slug"]
+
+
+class DirectoryUserEmail:
+    def __init__(self, attributes: JsonDict) -> None:
+        self.type: Optional[str] = attributes.get("type")
+        self.value: Optional[str] = attributes.get("value")
```

Comment:
Is value ever null/None based on our API schema?

Reply:
Yeah, I left a comment about this in the docs PR as well.